--- a/dotfiles/.config/zsh/.zshrc
+++ b/dotfiles/.config/zsh/.zshrc
@@ -37,6 +37,9 @@
 alias vim='nvim'
 alias la="ls -a"
 alias g="git"
+alias extra="sudo pacman -Rs $(pacman -Qqtd)"
+alias update="sudo pacman -Syu"
+alias install="sudo pacman -S"
 
 # Load syntax highlighting; should be last.
 source /usr/share/zsh/plugins/fast-syntax-highlighting/fast-syntax-highlighting.plugin.zsh 2>/dev/null
